sdcm/cluster.py

```diff
@@ -2568,28 +2569,20 @@ def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int
         with self.remote_scylla_yaml() as scylla_yml:
             scylla_yml.murmur3_partitioner_ignore_msb_bits = murmur3_partitioner_ignore_msb_bits
 
-        search_reshard_start = self.follow_system_log(patterns=[DB_LOG_PATTERN_RESHARDING_START])
-        search_reshard_finish = self.follow_system_log(patterns=[DB_LOG_PATTERN_RESHARDING_FINISH])
-        start_scylla_timeout = 7200
-        self.start_scylla(timeout=start_scylla_timeout)
-        resharding_started = list(search_reshard_start)
-        resharding_finished = list(search_reshard_finish)
+        resharding_start_ctx = f'Resharding started on node {self.name} with \
```

Comment:
no need to mention 'resharding started...' just giving `resharding_details=f"Resharding: (murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits})"`
which could be the same for both start and stop.
And later in error msg just append (to the end with some prefix like `<original exc mst>. Context: {err_msg_context}`